File: src/main/java/net/hectus/neobb/turn/legacy_game/structure/glass_wall/LTLightBlueGlassWall.java
package net.hectus.neobb.turn.legacy_game.structure.glass_wall;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.structure.PlacedStructure;
import net.hectus.neobb.turn.default_game.attributes.clazz.ColdClazz;
import net.hectus.neobb.turn.default_game.structure.glass_wall.GlassWallTurn;

public class LTLightBlueGlassWall extends GlassWallTurn implements ColdClazz {
    public LTLightBlueGlassWall(NeoPlayer player) { super(player); }
    public LTLightBlueGlassWall(PlacedStructure data, NeoPlayer player) { super(data, player); }

    @Override
    public String color() {
        return "light_blue";
    }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/legacy_game/structure/glass_wall/LTLightBlueGlassWall.kt
package net.hectus.neobb.modes.turn.legacy_game.structure.glass_wall

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.matrix.structure.PlacedStructure
import net.hectus.neobb.matrix.structure.StaticStructure
import net.hectus.neobb.matrix.structure.StaticStructures
import net.hectus.neobb.modes.turn.default_game.attribute.clazz.NeutralClazz
import net.hectus.neobb.modes.turn.default_game.structure.glass_wall.GlassWallTurn
import net.hectus.neobb.player.NeoPlayer

class LTLightBlueGlassWall(data: PlacedStructure?, cord: Cord?, player: NeoPlayer?) : GlassWallTurn(data, cord, player), NeutralClazz {
    override val staticStructure: StaticStructure = StaticStructures.Legacy.GlassWall.LIGHT_BLUE
}
